samples-java/build.gradle

```diff
@@ -8,6 +8,8 @@ android {
         minSdkVersion 21
         targetSdkVersion rootProject.ext.compileSdkVersion
 
+        consumerProguardFiles 'proguard-rules.pro'
```

Comment:
What is the purpose of this variable?

Reply:
Gson Will Crash at Enum Fields When Enabling Proguard. We should avoid Enum confusion, and  `consumerProguardFiles ` can make the SDK include its own ProGuard configuration file.